src/CosmosDB/CosmosDB/Az.CosmosDB.psd1

```diff
@@ -79,14 +79,30 @@ CmdletsToExport = 'Get-AzCosmosDBSqlContainer', 'Get-AzCosmosDBSqlContainerThrou
                'Get-AzCosmosDBSqlStoredProcedure', 'Get-AzCosmosDBSqlTrigger', 'Get-AzCosmosDBSqlUserDefinedFunction',
                'Set-AzCosmosDBSqlStoredProcedure', 'Set-AzCosmosDBSqlTrigger', 'Set-AzCosmosDBSqlUserDefinedFunction',
                'Remove-AzCosmosDBSqlStoredProcedure', 'Remove-AzCosmosDBSqlTrigger', 'Remove-AzCosmosDBSqlUserDefinedFunction',
-               'New-AzCosmosDBSqlConflictResolutionPolicy', 'New-AzCosmosDBSqlIndexingPolicy',
```

Comment:
this cmdlet "New-AzCosmosDBSqlConflictResolutionPolicy" was removed, seems like breaking change to me